dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/PermissionsWrapper.java

```diff
@@ -155,6 +156,10 @@ public boolean canViewUnpublishedDataset(DataverseRequest dr, Dataset dataset) {
     public boolean canUpdateDataset(DataverseRequest dr, Dataset dataset) {
         return doesSessionUserHaveDataSetPermission(dr, dataset, Permission.EditDataset);
     }
+    public boolean canCurrentUserUpdateDataset(Dataset dataset) {
```

Comment:
Because doesSessionUserHaveDataSetPermission requires for me to pass `DataverseRequest` and it seems that this class is designed to already be aware of logged user. To go further with this I don't see a point to have methods here that take dataverseRequest param.